### PR TITLE
Building with Active Record Micro Reddit

### PR DESCRIPTION
**Pull request by barackm**
## For this project I have completed the following:

- Setup linters
- Create the User model
- Create the Post model
- Create the Comment model
- Build associations between all the models
- Implement validation for all the models to get the correct data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,45 @@
...
+## Built With
 
-* Configuration
+- Ruby on Rails
 
-* Database creation
+## Getting Started
 
-* Database initialization
+**To get the local coppy of this project you just need to clone it by running the following command from your terminal `git clone https://github.com/barackm/micro-reddit.git`**
```
Comment (rindrajosia) [inline README.md]: please, change **coppy** to **copy**

File: README.md [hunk 2]
```diff
@@ -1,24 +1,45 @@
...
-* How to run the test suite
+# Installation
 
-* Services (job queues, cache servers, search engines, etc.)
+After you have cloned the repository, you need to install all the available gems by running the command `bundle install` from the root of the project.
 
-* Deployment instructions
+### Setup
 
-* ...
```
Comment (rindrajosia) [inline README.md:24]: Here you should ask user to create DB via **rails db:create** and migrate it via **rails db:migrate**

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- A user can be created :+1: 
- A post can be created :100: 
- A comment can be created :1st_place_medal: 

To improve :factory_worker: [REQUIRED]:

- [ ] Please check your readme by using tools like grammarly because it is the first thing user checks. (check inline comments below :arrow_double_down:)

Nice to have [OPTIONAL] :

- [ ]  

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Review by taiwo2 — APPROVED** [review]
# STATUS: APPROVED
@barackm  great job and thanks for working on your Readme.
 kindly add `rails` to your db:create as your readme talks alots about your professionalism

**Pull request merged**